patches/bbp-virtualenv/virtualenv-extra.py

```diff
@@ -25,3 +32,24 @@ def after_install(options, home_dir):
     if not osp.exists(manylinux_mod):
         with open(manylinux_mod, 'w') as ostr:
             ostr.write('manylinux1_compatible = True\n')
+
+    # wrap around pip to call patch_wheels after each call
+    pip_bin = osp.join(home_dir, 'bin', 'pip')
+    os.rename(pip_bin, pip_bin + '.bin')
```

Comment:
virtualenv creates several pip executable, for instance pip, pip2, and pip2.7
You should patch all of them. Furthermore, I would prefix the original pip scripts by . to hide them from the user.

Reply:
This is done through a symlink in the lines after ;)
However this breaks pip updates so I am changing to:
- wrappers in bin-wrap
- patch PATH in activate to add bin-wrap before bin

Thoughts ?